tests/documentation/command-reference/doc_command_reference_createnamespace_test.go

```diff
@@ -0,0 +1,64 @@
+package docautomation
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/go-cmp/cmp"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-developer/odo/tests/helper"
+)
+
+var _ = Describe("doc command reference odo create namespace", func() {
+	var commonVar helper.CommonVar
+	var commonPath = filepath.Join("command-reference", "docs-mdx", "create-namespace")
+	var outputStringFormat = "```console\n$ odo %s\n%s```\n"
+
+	BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+		helper.Chdir(commonVar.Context)
+		Expect(helper.VerifyFileExists(".odo/env/env.yaml")).To(BeFalse())
+	})
+
+	AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+
+	Context("To create a namespace resource", func() {
```

Comment:
Perhaps you can change the namespace that is created to something more generic, and then move both the tests to a single context like you did before, that might make this more simple. WDYT?

Reply:
Yeah I have committed the changes

Reply:
I think you might not have pushed them? Can you check again?